docs/install/operator/knative-with-operator-cli.md

```diff
@@ -8,64 +8,22 @@ function: how-to
 
 # Install by using the Knative Operator CLI Plugin
 
-Knative provides a CLI Plugin to install, configure and manage Knative via the command lines. This CLI plugin facilitates
-you with a parameter-driven way to configure the Knative cluster, without interacting with the complexities of the custom
-resources.
+Knative provides the Operator CLI Plugin to install, configure and manage Knative on the command line. This plugin helps you configure the Knative cluster, without interacting with the complexities of the custom resources.
 
---8<-- "prerequisites.md"
---8<-- "security-prereqs-binaries.md"
+This topic describes how to install the Knative Operator and the Serving and Eventing components using a CLI. To install the manifests and other resources, see [Knative Operator](knative-with-operators.md).
```

Comment:
```suggestion
This topic describes how to install the Knative Operator and the Serving and Eventing components using a CLI.
```

I think `kn operator install` will install the manifests for you.  Some people like to be able to see what they are installing, while other people like the one-command experience.